.github/pull_request_template.md

```diff
@@ -1 +1,22 @@
-Github issue / Community forum post (link here to close automatically):
+## Summary
+Provide details about your pull request and what it adds, fixes, or changes. Photos and videos are recommended.
+...
+
+#### How to test the change:
+1. ...
+
+
+## Issues fixed
+Include links to Github issue or Community forum post or **Linear ticket**:
+> Important in order to close automatically and provide context to reviewers
+
+...
+
+
+## Review / Merge checklist
+- [ ] PR title and summary are descriptive. **Remember, the title automatically goes into the changelog. Use `(no-changelog)` otherwise.**
```

Comment:
Idea for a folllow up improvent: Document somewhere when should we use `feat`, `fix`, `chore`, etc.

Reply:
done. exported from notion